pyout.py

```diff
@@ -33,7 +30,7 @@ class Tabular(object):
     style : dict, optional
         Each top-level key should be a column name and the value
         should be a style dict that overrides the `default_style`
-        class attribute.
+        class attribute.  See the "Examples" section below.
```

Comment:
Examples are good. But may be add a module level constant `STYLE_ATTRIBUTES` listing all the allowed attributes (color, bold, ...)? and then include (or reference) here?

Reply:
right, definitely need some sort of definition of all possible attributes.  Figured I'd wait until some of the style stuff stabilized, but I can do that now if you'd like.

Reply:
> module level constant STYLE_ATTRIBUTES

Thinking about this more: A list (or set) of allowed attributes would
be useful if/when we want to barf if we get an unknown attribute, but
it terms of documentation, it wouldn't help describe the acceptable
values, etc.  Seems like we should just have one place where we
document the style specification (not sure where's best.. module
docstring?) and refer to it in all other places.

Reply:
agreed!  but where that place will be? eventually it should be reflected in the docs, but also in the code.
ideally it should be "functional", i.e. actual values verified against it etc. So may be `STYLE_ATTRIBUTES` could be started as a dict (eventually become some kind of full json schema?) of type
```python
STYLE_ATTRIBUTES = dict(
 color=dict(
   description="The color",
   types=(str, unicode, ...),
 ),  
 ...
)
```
? this way you could not only use it to verify but auto-compose text listing this stuff etc

Reply:
noted: #18

yeah, it's certainly nice in terms of avoiding stale documentation.
Can't decide if having a spec for the spec is an overkill :)